tests/src/core/Polyfill.test.ts

```
@@ -38,36 +38,65 @@
     };
-    
+
     const arrayBuffer = new ArrayBuffer(10);
     const result = await context.decodeAudioData(arrayBuffer);
     expect(result).to.have.property("duration", 10);
   });
+
+  it("TextMetrics", async () => {
+    if (window.TextMetrics) {
+      // @ts-ignore
+      delete TextMetrics.prototype.actualBoundingBoxLeft;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)